tests/sample/xdp_common.h

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#include "bpf_helpers.h"
+#include "ebpf.h"
+#include "xdp_tests_common.h"
+
+void
+swap_mac_addresses(ETHERNET_HEADER* ethernet_header)
+{
+    mac_address_t mac = {0};
+    memcpy(mac, ethernet_header->Destination, sizeof(mac_address_t));
+    memcpy(ethernet_header->Destination, ethernet_header->Source, sizeof(mac_address_t));
+    memcpy(ethernet_header->Source, mac, sizeof(mac_address_t));
+}
```

Comment:
Function bodies should be in .c files not .h files

Reply:
This is so that 2 eBPF programs in 2 different .c files can share code. CLANG inlines these. I have seen many examples of such uses in eBPF samples. Would it help if I explicitly marked them inline?

Reply:
Yes, it would help to mark them inline